bigchaindb/core.py

```diff
@@ -433,14 +387,17 @@ def get_owned_ids(self, owner):
         """
         return self.get_outputs_filtered(owner, include_spent=False)
 
+    @property
```

Comment:
Can we make this so that we don't need to instantiate a new `FastQuery` instance every time we need to use it, or move it under `__init__`?